order/src/main/java/com/caldevsupplychain/order/service/OrderServiceImpl.java

```diff
@@ -83,38 +135,25 @@ public OrderBean deleteOrder(OrderBean orderBean) {
 	}
 
 	@Override
-	public Optional<List<OrderBean>> findByDisplayId(String displayId) {
-		List<Order> orders = orderRepository.findByDisplayId(displayId, new PageRequest(0, Integer.MAX_VALUE));
-		if(orders != null){
-			return Optional.of(orderMapper.ordersToBeans(orders));
-		}
-		return Optional.empty();
-	}
+	public Optional<List<OrderBean>> findByUserUuid(String userUuid) {
 
-	@Override
-	public Optional<List<OrderBean>> findByUserId(String userId) {
-		List<Order> orders = orderRepository.findByUserId(userId, new PageRequest(0, Integer.MAX_VALUE));
+		List<Order> orders = orderRepository.findByUserUuid(userUuid, new PageRequest(0, Integer.MAX_VALUE));
 
 		if(orders != null){
-			return Optional.of(orderMapper.ordersToBeans(orders));
+			return Optional.of(orderMapper.ordersToBeans(orders, new CycleAvoidingMappingContext()));
 		}
 		return Optional.empty();
 	}
 
 	@Override
-	public Optional<List<OrderBean>> findByAgentId(String agentId) {
-		List<Order> orders = orderRepository.findByAgentId(agentId, new PageRequest(0, Integer.MAX_VALUE));
+	public Optional<List<OrderBean>> findByAgentUuid(String agentUuid) {
```

Comment:
return `List<OrderBean>` instead of `Optional<List<OrderBean>>`

Reply:
Fixed